dist/weather-card.js

```diff
@@ -168,7 +168,7 @@ class WeatherCard extends LitElement {
           style="background: none, url('${this.getWeatherIcon(
             stateObj.state.toLowerCase(),
             this.hass.states["sun.sun"]
-          )}') no-repeat; background-size: contain;"
+          )}') no-repeat; background-size: contain; image-rendering: optimizeQuality"
```

Comment:
I wonder why we need to do this since SVG is supposed to scale.

Reply:
Have a look at the pull request. I Tested it and this won't help but I hope someone can help to close the request